docs/content/en/docs/tasks/troubleshoot/_supportbundle.md

```diff
@@ -6,53 +6,53 @@ description: >
     Using the Support Bundle with your EKS Anywhere Cluster
 ---
 
-This guide covers the use of the EKS Anywhere Support Bundle for troubleshooting and support.
+This guide covers the use of the EKS Anywhere support bundle for troubleshooting and support.
 This allows you to gather cluster information, save it to your administrative machine, and perform analysis of the results.
 
-EKS Anywhere leverages [troubleshoot.sh](https://troubleshoot.sh/) to [collect](https://troubleshoot.sh/docs/collect/) and [analyze](https://troubleshoot.sh/docs/analyze/) kubernetes cluster logs, 
+EKS Anywhere leverages [troubleshoot.sh](https://troubleshoot.sh/) to [collect](https://troubleshoot.sh/docs/collect/) and [analyze](https://troubleshoot.sh/docs/analyze/) Kubernetes cluster logs, 
 cluster resource information, and other relevant debugging information. 
 
-EKS Anywhere has two Support Bundle commands:
+EKS Anywhere has two support bundle commands:
 
 `eksctl anywhere generate support-bundle` will execute a support bundle on your cluster, 
 collecting relevant information, archiving it locally, and performing analysis of the results.
 
-`eksctl anywhere generate support-bundle-config` will generate a Support Bundle config yaml file for you to customize.
+`eksctl anywhere generate support-bundle-config` will generate a support bundle config YAML file for you to customize.
 
-### Collecting a Support Bundle and running analyzers
+### Collecting a support bundle and running analyzers
 ```
 eksctl anywhere generate support-bundle
 ```
 
-`generate support-bundle` will allow you to quickly collect relevant logs and cluster resources and save them locally in an archive file.
+`generate support-bundle` will allow you to collect relevant logs and cluster resources quickly and save them locally in an archive file.
 This archive can then be used to aid in further troubleshooting and debugging.
 
 If you provide a cluster configuration file containing your cluster spec using the `-f` flag,
 `generate support-bundle` will customize the auto-generated support bundle collectors and analyzers 
 to match the state of your cluster.
 
-If you provide a support bundle configuration file using the `--bundle-config` flag, 
-for example one generated with `generate support-bundle-config`, 
+If you provide a support bundle configuration file using the `--bundle-config` flag
+(for example one generated with `generate support-bundle-config`), 
 `generate support-bundle` will use the provided configuration when collecting information from your cluster and analyzing the results.
 
 ```
 Flags:
-      --bundle-config string   Bundle Config file to use when generating support bundle
```

Comment:
Not a huge deal, but since these are output snippets from the binary itself, I feel these changes should come only after the tool has been updated to produce these outputs.

Reply:
Do you mean all the proposed changes under `flags`?

Reply:
Fixed in next revision